source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -1620,6 +1620,7 @@ export class TextInput extends Input {
     maxLength: number;
     isMultiline: boolean;
     placeholder: string;
+    style: Enums.InputTextStyle = Enums.InputTextStyle.Text;
```

Comment:
This won't work, Enums.InputTextStyle is a class, and the Text field is a string, can't assign string to class.

InputTextStyle needs to be defined as an enumeration like pretty much everything else in enums.ts

Reply:
Funny how Github can't track the fact that the class was changed on a different file and that fulfills your request; would you need to remove the feedback @dclaux ?